content/blogs/Fuzzing-ReactOS.md

```diff
@@ -0,0 +1,43 @@
+---
+title:       "Syzkaller - Fuzzing ReactOS in 2020"
+author:      "Freakston"
+date:        2020-08-04
+tags:        [ "gsoc", "gsoc-2020" ]
+---
+
+Hello everyone. In this blog post, I will be talking about the steps to set up Syzkaller for ReactOS locally and start fuzzing. 
+If you have been following the blog posts on the ReactOS website then you would have noticed that this is my Google Summer of Code 2020 Project. 
+Now that Coding period 2 has officially come to an end and the work is nearing completion, this post will help others in setting up the fuzzer get started with fuzzing. 
+
+If you haven't checked out the posts regarding the progress, here they are -
+
+[PORTING SYZKALLER TO REACTOS](https://reactos.org/blogs/porting-syzkaller-to-reactos-gsoc-2020/)
+
+[PORTING SYZKALLER TO REACTOS (EVALUATION-1)](https://reactos.org/blogs/porting-syzkaller-to-reactos-1-gsoc2020/)
```

Comment:
Suggesting relative links instead of absolute
```suggestion
[PORTING SYZKALLER TO REACTOS](/blogs/porting-syzkaller-to-reactos-gsoc-2020/)

[PORTING SYZKALLER TO REACTOS (EVALUATION-1)](/blogs/porting-syzkaller-to-reactos-1-gsoc2020/)
```